docs/sources/guides/whats-new-in-v7-0.md

```diff
@@ -186,6 +186,14 @@ This release includes a series of features that build on our new usage analytics
 
 {{< docs-imagebox img="/img/docs/v70/presence_indicator.jpg" max-width="1024px" caption="Grafana Enterprise - Presence Indicator" >}}
 
+### SAML Role and Team Sync
+
+SAML support in Grafana Enterprise is improved by adding Role and Team Sync. Read more about how to use these features in the [documentation]({{< relref "../enterprise/saml.md#configure-team-sync" >}}).
```

Comment:
For people using screen readers, it's always a good idea to make sure the link isn't too generically sounding.

```suggestion
SAML support in Grafana Enterprise is improved by adding Role and Team Sync. Read more about how to use these features in the [SAML team sync documentation]({{< relref "../enterprise/saml.md#configure-team-sync" >}}).
```

Reply:
Good point!